### PR TITLE
docker login changes

### PR DESCRIPTION
**Pull request by tanvigour**
Signed-off-by: tanvigour <tanvi.gour@gmail.com>

# Description

Login to Docker

## Issue reference

We strive to have all PRs being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] The quickstart code compiles correctly
* [ ] You've tested new builds of the quickstart if you changed quickstart code
* [ ] You've updated the quickstart's README if necessary
* [ ] If you have changed the steps for a quickstart be sure that you have updated the automated validation accordingly. All of our quickstarts have annotations that allow them to be executed automatically as code. For more information see [mechanical-markdown](https://github.com/dapr/mechanical-markdown#readme). For user guide with examples see [Examples](https://github.com/dapr/mechanical-markdown/tree/main/examples#mechanical-markdown-by-example).

### CODE REVIEW

File: .github/workflows/docker_login.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: Docker login
+
+on:
+  push:
+    branches:
+      - master
+      - tanvi/new_branch
```
Comment (artursouza) [inline .github/workflows/docker_login.yml:12]: Please, remove your branch from the GH file.

File: .github/workflows/docker_login.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+jobs:
+  build:
+    name: docker login
+    runs-on: ubuntu-latest
```
Comment (artursouza) [inline .github/workflows/docker_login.yml:17]: Why are you creating a new workflow? You probably need to change the existing workflow to do a login before doing docker pulls.

**Review by artursouza — APPROVED** [review]

**Review by artursouza — CHANGES_REQUESTED** [review]
Requested changes. Wrong button, sorry

File: .github/workflows/docker_login.yml (hunk 1 above)
Comment (tanvigour) [reply .github/workflows/docker_login.yml:12]: The idea was to keep this specifically for docker account monitoring. For ex, doing login, adding rate check etc. 

But now that I see this : https://github.com/dapr/quickstarts/blob/feature/new_quickstarts/.github/workflows/build.yml looks like build workflow does have docker login in it, maybe now it makes more sense to check the limit instead cause maybe we are running into pull limits

**Pull request closed without merging**